src/main/target/TINYFISH/target.h

```diff
@@ -66,7 +66,7 @@
 #define UART3_TX_PIN            PB10
 #define UART3_RX_PIN            PB11
 
-#define SBUS_TELEMETRY_UART     SERIAL_PORT_USART2
+#define SBUS_AND_TELEMETRY_UART SERIAL_PORT_USART2
```

Comment:
I'd rather this was done by setting `SERIAL_PORT_USART2` directly in `config.c`. Cleaning up stale defines from `target.h` is hard enough as it is, having target specific defines in there makes it even harder, and in reality `config.c` will always hold config information as well.